xl_auth/templates/users/administer.html

```diff
@@ -0,0 +1,28 @@
+<!-- users/administer.html -->
```

Comment:
Yeah, I'm not fond of the current name either. `/admin` would be an easy way of separating things, but that moves us even farther away from a REST-ish API. :(

Let's leave it as is for now, but I agree that it would be very nice to clean this up soon.